### PR TITLE
zile 2.6.1

### PR DESCRIPTION
**Pull request by carlocab**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/zile.rb [hunk 1]
```diff
@@ -21,6 +21,8 @@ class Zile < Formula
   depends_on "help2man" => :build
   depends_on "pkg-config" => :build
   depends_on "bdw-gc"
+  depends_on "glib"
+  depends_on "libgee"
```
Comment (Rylan12) [inline Formula/zile.rb:25]: Are these build-only? From the `NEWS` file:

```
** Miscellaneous

  Zile has been rewritten in Vala. This makes the code considerably shorter
  (~6kLoC as compared with 8kLoC for the C version) and easier to
  maintain. It now requires glib and libgee to build, but a Vala compiler is
  not required, as the generated C sources are shipped in release tarballs.
```

Comment (carlocab) [reply Formula/zile.rb:25]: Oops. I did read the `NEWS` file, but I missed that part.

Comment (Rylan12) [reply Formula/zile.rb]: honestly ignoring the things that I just read in the instructions is the source of most of my problems...

**Comment by carlocab** [comment]
Looks like `glib` is needed at runtime:
```
==> /opt/homebrew/Cellar/zile/2.6.1/bin/zile --version
dyld: Library not loaded: /opt/homebrew/opt/glib/lib/libgio-2.0.0.dylib
  Referenced from: /opt/homebrew/Cellar/zile/2.6.1/bin/zile
  Reason: image not found
```

**Comment by Rylan12** [comment]
Yeah. Looks like maybe both based on:

```
==> brew linkage --test zile
==> FAILED
Broken dependencies:
  /opt/homebrew/opt/gettext/lib/libintl.8.dylib (gettext)
  /opt/homebrew/opt/glib/lib/libgio-2.0.0.dylib (glib)
  /opt/homebrew/opt/glib/lib/libglib-2.0.0.dylib (glib)
  /opt/homebrew/opt/glib/lib/libgobject-2.0.0.dylib (glib)
  /opt/homebrew/opt/libgee/lib/libgee-0.8.2.dylib (libgee)
```

**Comment by carlocab** [comment]
Ah, yes. Looks like it's both indeed.

**Review by Rylan12 — APPROVED** [review]

**Pull request closed without merging**